instrumentation/r2dbc-postgresql-0.9.2/build.gradle

```diff
@@ -10,8 +10,8 @@ jar {
 }
 
 verifyInstrumentation {
-    passesOnly 'org.postgresql:r2dbc-postgresql:[0.9.2,1.0.0.RELEASE)'
-    excludeRegex(".*(M1|M2|RC).*")
+    passesOnly 'org.postgresql:r2dbc-postgresql:[0.9.2,)'
+    exclude 'org.postgresql:r2dbc-postgresql:1.0.0.RC1'
```

Comment:
Something was odd with this release.
Removed the regex so if there is an incompatibility with any preview of 2.0.0 we will be notified then, instead of when the final version is released.